packages/contentful--create-contentful-app/src/template.ts

```diff
@@ -65,15 +74,15 @@ function cleanUp(destination: string) {
   rmIfExists(resolve(destination, 'yarn.lock'));
 }
 
-export async function cloneTemplateIn(destination: string, options: CLIOptions) {
-  const source = getTemplateSource(options);
+export async function cloneExampleIn(destination: string, options: CLIOptions) {
```

Comment:
Also here I would use template, for the saem reason as above

Reply:
Yeah, makes sense.